src/native/libs/System.Globalization.Native/static-build/build-nativeaot-shim-standalone.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
Nit: This script can be called just `build.sh`. The directory name is  enough to disambiguate it from other `build.sh` scripts.